docker/Dockerfile

```diff
@@ -12,7 +14,7 @@ RUN apt-get update \
         ca-certificates \
         adduser \
     # Install google-fluentd at version 1.6.0-1.
```

Comment:
Let's move this comment above the `ARG` (and change `Install` to `Default to`.

Reply:
Done.